### ISSUE TITLE
convert_lines_to_text() missing 1 required positional argument

### ISSUE THREAD

**Issue by ghost** [JonathanRaiman/epub_conversion#2]
Hi, I have an issue and I don't know how to fix it.

`converter.convert("my_succinct_text_file.gz")
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "C:\Users\lguic\Desktop\TIPE\EPUB.py", line 98, in convert
    for sentence in convert_lines_to_text(convert_epub_to_lines(book)):
TypeError: convert_lines_to_text() missing 1 required positional argument: 'article_title'
`

Thank you for your help 💯

**Comment by padmalcom** [comment]
Same here :)

**Comment by ricest** [comment]
Me three. Is it enough to just get rid of the extra argument? I can't see it in use:

```
def convert_lines_to_text(lines, article_title):
	for sentence in to_raw_text(lines):
		yield " ".join(sentence)+"\n"
```

**Comment by MagedSaeed** [comment]
me too

**Comment by JonathanRaiman** [comment]
Great catch. Should be fixed in 1.0.8, that argument is unnecessary
